renderer/components/Onboarding/Steps/Recover.js

```diff
@@ -1,26 +1,134 @@
-/* eslint-disable react/no-multi-comp */
-
 import React from 'react'
 import PropTypes from 'prop-types'
+import Downshift from 'downshift'
 import { compose } from 'redux'
 import { withFormApi } from 'informed'
 import { FormattedMessage, injectIntl } from 'react-intl'
 import bip39 from 'bip39-en'
-import { Flex } from 'rebass/styled-components'
+import { Box, Flex } from 'rebass/styled-components'
+import styled, { withTheme } from 'styled-components'
+import { themeGet } from '@styled-system/theme-get'
 import { intlShape } from '@zap/i18n'
 import parseSeed from '@zap/utils/parseSeed'
-import { Bar, Header } from 'components/UI'
+import { Bar, Header, Text } from 'components/UI'
 import { Form, Input, Label } from 'components/Form'
 import messages from './messages'
 
+const StyledOptionList = styled.ul`
+  padding: 0;
+  margin-top: 4px;
+  position: absolute;
+  z-index: 2;
+  width: 100%;
+  max-height: 300px;
+  overflow-y: auto;
+  overflow-x: hidden;
+  outline: 0;
+  transition: opacity 0.1s ease;
+  background-color: ${themeGet('colors.secondaryColor')};
+  border-radius: 5px;
+  box-shadow: ${themeGet('shadows.s')};
+`
+
+const SeedWordList = props => {
+  const { bip39WordMap, highlightedIndex, getItemProps, getMenuProps, inputValue, theme } = props
+
+  if (!inputValue) {
+    return null
+  }
+
+  const wordsToFilter = bip39WordMap[inputValue[0]]
+  if (!wordsToFilter) {
+    return null
+  }
+
+  const wordList = wordsToFilter.filter(item => item.indexOf(inputValue) === 0)
```

Comment:
`item.indexOf(inputValue) < 0` won't work as that would find items that do not match the input value. `.indexOf(inputValue) === 0` matches the input value string exactly from the beginning. So, for example, if a user input `app` the word `apple` would match since `app` starts at the zero index of `apple`.

I will work on this to make it more obvious what's happening here.

Reply:
Tried to make it more clear. Should be better now.